src/rgw/rgw_tag.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RGW_TAG_H
+#define RGW_TAG_H
+
+#include <map>
+#include <string>
+#include <include/types.h>
+
+#include "rgw_common.h"
+
+using std::string;
```

Comment:
@chardan, @theanalyst agree, and we've stopped doing this for the most part, I think you should just use std::string in the header